ibis/backends/flink/ddl.py

```diff
@@ -107,6 +128,15 @@ def __init__(
         self.temporary = temporary
         self.watermark = watermark
 
+        self.primary_keys = set(primary_key.split(",")) if primary_key else None
+        # Check if `primary_keys` is a subset of the columns in `schema`.
+        if self.primary_keys and not self.primary_keys <= set(schema.names):
+            raise exc.IbisError(
+                "`primary_keys` must be a subset of the columns in `schema`. \n"
```

Comment:
nit: `primary_keys` -> `primary_key` because that's the name of the argument you're using

Reply:
Thanks, done.